### ISSUE TITLE
`yarn lint` throws dependency errors

### ISSUE THREAD

**Issue by bartaz** [canonical-web-and-design/react-components#264]
I can't locally run lint.
`yarn lint` throws 

```
Error: Failed to load plugin 'jsx-a11y' declared in '.eslintrc.js » eslint-config-react-app': Cannot find module '../internals/wrapped-well-known-symbol'
```

<img width="1403" alt="Screenshot 2020-09-04 at 16 04 22" src="https://user-images.githubusercontent.com/83575/92248000-53802b80-eec8-11ea-9cea-6fe37b23881f.png">

I tried reinstalling with `yarn lint`, but it doesn't seem to change anything.

**Comment by squidsoup** [comment]
Interesting, I can't reproduce this on master.

**Comment by bartaz** [comment]
Same on master:

```
~/Canonical/react-components [master] $ yarn lint
yarn run v1.7.0
$ yarn lint-package-json && yarn lint-js && yarn lint-style
$ npmPkgJsonLint .
$ eslint src

Oops! Something went wrong! :(

ESLint: 7.8.0

Error: Failed to load plugin 'jsx-a11y' declared in '.eslintrc.js » eslint-config-react-app': Cannot find module '../internals/wrapped-well-known-symbol'
Require stack:
- /Users/bartaz/Canonical/react-components/node_modules/core-js-pure/internals/define-well-known-symbol.js
- /Users/bartaz/Canonical/react-components/node_modules/core-js-pure/modules/es.symbol.js
- /Users/bartaz/Canonical/react-components/node_modules/core-js-pure/es/symbol/index.js
- /Users/bartaz/Canonical/react-components/node_modules/core-js-pure/stable/symbol/index.js
- /Users/bartaz/Canonical/react-components/node_modules/@babel/runtime-corejs3/core-js-stable/symbol.js
- /Users/bartaz/Canonical/react-components/node_modules/aria-query/lib/rolesMap.js
- /Users/bartaz/Canonical/react-components/node_modules/aria-query/lib/index.js
- /Users/bartaz/Canonical/react-components/node_modules/eslint-plugin-jsx-a11y/lib/rules/aria-activedescendant-has-tabindex.js
- /Users/bartaz/Canonical/react-components/node_modules/eslint-plugin-jsx-a11y/lib/index.js
- /Users/bartaz/Canonical/react-components/node_modules/eslint/lib/cli-engine/config-array-factory.js
- /Users/bartaz/Canonical/react-components/node_modules/eslint/lib/cli-engine/cascading-config-array-factory.js
- /Users/bartaz/Canonical/react-components/node_modules/eslint/lib/cli-engine/cli-engine.js
- /Users/bartaz/Canonical/react-components/node_modules/eslint/lib/eslint/eslint.js
- /Users/bartaz/Canonical/react-components/node_modules/eslint/lib/eslint/index.js
- /Users/bartaz/Canonical/react-components/node_modules/eslint/lib/cli.js
- /Users/bartaz/Canonical/react-components/node_modules/eslint/bin/eslint.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1029:15)
    at Function.Module._load (internal/modules/cjs/loader.js:898:27)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
    at require (/Users/bartaz/Canonical/react-components/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at Object.<anonymous> (/Users/bartaz/Canonical/react-components/node_modules/core-js-pure/internals/define-well-known-symbol.js:3:36)
    at Module._compile (/Users/bartaz/Canonical/react-components/node_modules/v8-compile-cache/v8-compile-cache.js:194:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by bartaz** [comment]
Hmm... I upgraded `yarn` and it fixed the issue...
